app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -777,6 +777,12 @@ interface ProjectViewModel {
                     .filter { it.project().isLive && !it.project().isBacking }
                     .compose(bindToLifecycle())
                     .subscribe { this.lake.trackCreatorDetailsClicked(it) }
+
+            projectData
+                    .compose<ProjectData>(takeWhen(this.fixPaymentMethodButtonClicked))
+                    .compose(bindToLifecycle())
+                    .subscribe{ this.lake.trackFixPledgeButtonClicked(it) }
```

Comment:
So now that we have a tracking event, we need to add a test! `testShowUpdatePledge_whenFixingPaymentMethod`

Reply:
I complety forgot about the test! sorry :anguished:

Reply:
Np, you got this 🚀